apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -58,6 +58,13 @@ public sealed class SpannerTransaction : DbTransaction, ISpannerTransaction
 
         private Session Session { get; }
 
+        // Note: We use seconds here to follow the convention set by DbCommand.CommandTimeout.
```

Comment:
We could make that explicit with a suffix? Inconsistent with DbCommand in terms of naming, but at least it's clearer. On the other hand, if we're going to be inconsistent anyway, we could make it a TimeSpan. Hmm. Not sure.